composer.json

```diff
@@ -49,7 +54,5 @@
     },
     "config": {
         "preferred-install": "dist"
-    },
-    "minimum-stability": "stable",
-    "prefer-stable": true
```

Comment:
No clue, I may have merged and accidentally cherry-picked my original file instead of yours (which had this). My bad.